<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50, 68, 103);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushStart">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>610</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(175, 144, 84);
font: 24pt &quot;.AppleSystemUIFont&quot;;
border-color: rgb(27, 35, 50);
border-style: outset;
border-width: 2px;
border-radius: 16px;</string>
    </property>
    <property name="text">
     <string>Start Simulation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushStatement">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(34, 51, 87);
font: 16pt &quot;.AppleSystemUIFont&quot;;
border-color: rgb(27, 35, 50);
border-style: outset;
border-width: 2px;
border-radius: 16px;</string>
    </property>
    <property name="text">
     <string>Company Statement</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushHelp">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(34, 51, 87);
font: 16pt &quot;.AppleSystemUIFont&quot;;
border-color: rgb(27, 35, 50);
border-style: outset;
border-width: 2px;
border-radius: 16px;</string>
    </property>
    <property name="text">
     <string>Get Help</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushUpdate">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>100</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(34, 51, 87);
font: 16pt &quot;.AppleSystemUIFont&quot;;
border-color: rgb(27, 35, 50);
border-style: outset;
border-width: 2px;
border-radius: 16px;</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushTeam">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(34, 51, 87);
font: 16pt &quot;.AppleSystemUIFont&quot;;
border-color: rgb(27, 35, 50);
border-style: outset;
border-width: 2px;
border-radius: 16px;</string>
    </property>
    <property name="text">
     <string>Meet The Team</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushVersion">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>100</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(34, 51, 87);
font: 16pt &quot;.AppleSystemUIFont&quot;;
border-color: rgb(27, 35, 50);
border-style: outset;
border-width: 2px;
border-radius: 16px;</string>
    </property>
    <property name="text">
     <string>Version</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>801</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);
border-style: outset;
border-width: 6px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="logo.qrc">:/logo/elevateLogo.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCompany_Statement">
   <property name="text">
    <string>Company Statement</string>
   </property>
  </action>
  <action name="actionGroup_Members">
   <property name="text">
    <string>Staff Information</string>
   </property>
  </action>
  <action name="actionContact_Us">
   <property name="text">
    <string>Contact Us</string>
   </property>
  </action>
  <action name="actionGet_Update">
   <property name="text">
    <string>Get Update</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
